src/hotspot/share/opto/superword.cpp

```diff
@@ -1814,31 +1805,88 @@ uint SuperWord::max_implemented_size(const Node_List* pack) {
   }
 }
 
-bool SuperWord::requires_long_to_int_conversion(int opc) {
-  switch(opc) {
-    case Op_PopCountL:
-    case Op_CountLeadingZerosL:
-    case Op_CountTrailingZerosL:
-      return true;
-    default:
-      return false;
```

Comment:
Note: moved to `VectorNode::is_scalar_op_that_returns_int_but_vector_op_returns_long`